index.d.ts

```diff
@@ -127,31 +127,27 @@ declare namespace apm {
     setSpanOutcome(outcome: Outcome): void;
   }
 
-  type Outcome = 'unknown' | 'success' | 'failure'
+  type Outcome = 'unknown' | 'success' | 'failure';
```

Comment:
Not necessary for this PR -- but seeing the added `;` makes me wonder if we want to get some sort of auto-fix linter looking at these typescript files as well.

Reply:
Yah, agreed it would be nice, but I didn't want to go there in this PR.